drivers/network/ndis/ndis/miniport.c

```diff
@@ -2399,7 +2399,26 @@ NdisIDispatchPnp(
         Irp->IoStatus.Information |= Adapter->NdisMiniportBlock.PnPFlags;
         break;
 
+      case IRP_MN_REMOVE_DEVICE:
+        Status = IoSetDeviceInterfaceState(&Adapter->NdisMiniportBlock.SymbolicLinkName, FALSE);
```

Comment:
@mrmks04 I would suggest you put the implementation of this IRP into a new
```c
NTSTATUS
NTAPI
NdisIRemoveDevice(
    IN PDEVICE_OBJECT DeviceObject,
    IN PIRP Irp)
```
function (implementation could go below that of `NdisIAddDevice` in this file).